homeassistant/components/renault/services.yaml

```diff
@@ -86,3 +86,14 @@ charge_start:
       selector:
         device:
           integration: renault
+          
+charge_pause:
```

Comment:
There is no need to add a service if there is a button entity for it.

Reply:
I want to call the stop charge service via automation. Can I do it with a button?